cmd/common.go

```diff
@@ -34,10 +34,12 @@ func createIdentity(ctx context.Context, idService identity.Service) (identity.C
 }
 
 func generateKeys(config config.Configuration) {
-	p2pPub, p2pPvt := config.GetSigningKeyPair()
+	p2pPub, p2pPvt := config.GetP2PKeyPair()
```

Comment:
Good! When I brought this up in the housekeeping meeting, I was precisely worried that in a place like this, we'd do mainAccount.GetP2PKeyPair().